services/smart-fields-values-injector.js

```diff
@@ -8,13 +8,18 @@ function SmartFieldsValuesInjector(record, modelName) {
   var schema = Schemas.schemas[modelName];
 
   function setSmartFieldValue(record, field, modelName) {
+    var value;
     if (field.value) {
       logger.warn('DEPRECATION WARNING: Smart Fields "value" method is ' +
         'deprecated. Please use "get" method in your collection ' +
         modelName + ' instead.');
     }
 
-    var value = field.get ? field.get(record) : field.value(record);
+    try {
+      value = field.get ? field.get(record) : field.value(record);
+    } catch (e) {
+      logger.error(e);
```

Comment:
I think you can be more explicit for the developer about what happened. It is good to send the stack trace but an explanation is even better.
Something like:
```javascript
logger.error('Cannot retrieve the ' + field.field + ' value because of an internal error in the getter implementation: ' + error);
```